static/src/javascripts/projects/common/modules/experiments/tests/facebook-like-prompt.js

```diff
@@ -0,0 +1,176 @@
+                    status: true, // check login status
+                    cookie: true, // enable cookies to allow the server to access the session
+                    xfbml: true // parse XFBML
+                });
+            });
+        };
+
+        var progressBarTest = function (editions, options) {
+            return function () {
+                var edition = editions[config.page.edition];
+
+                getLikes(username)
+                    .then(function (likes) {
+                        var remainingLikes = edition.target - likes;
+                        var percentageComplete = (likes / edition.target) * 100;
+
+                        new Message('facebook-like-prompt', {
+                            pinOnHide: false,
+                            siteMessageCloseBtn: 'hide'
+                        }).show(getFacebookLikePromptProgressBarTemplate(edition, likes, remainingLikes, percentageComplete, options));
+
+                        loadFacebookSdk();
+                    });
+            };
+        };
+
+        this.variants = [
+            {
+                id: 'control',
+                test: function () {}
+            },
+            {
+                id: 'simple',
+                test: function () {
+                    new Message('facebook-like-prompt', {
+                        pinOnHide: false,
+                        siteMessageCloseBtn: 'hide'
+                    }).show(template(
+                        '<p class="site-message__message" id="site-message__message">{{messageText}}</p>' +
+                        '<div class="site-message--facebook-like-prompt__like">' +
+                            '<div class="fb-like"' +
+                                 'data-href="https://www.facebook.com/' + username + '"' +
+                                 'data-layout="button_count"' +
+                                 'data-action="like"' +
+                                 'data-show-faces="true"' +
+                                 'data-share="false"></div>' +
+                        '</div>',
+                        {
+                            messageText: 'Like the Guardian on Facebook'
+                        }
+                    ));
+
+                    loadFacebookSdk();
+                }
+            },
+            {
+                id: 'progress-bar--high-target',
+                test: progressBarTest({
+                    'UK': { target: 10000000, textTarget: '10 million' },
+                    'AU': { target: 500000 },
+                    'US': { target: 500000 }
+                }, { showIncompleteLabelInsideProgressBar: true })
+            },
+            {
+                id: 'progress-bar--low-target',
+                test: progressBarTest({
+                    'UK': { target: 5000000, textTarget: '5 million' },
+                    'AU': { target: 400000 },
+                    'US': { target: 300000 }
+                })
```

Comment:
Lets do this:

US is a bit more extreme, but there's certainly more growth potential there.

{
+                id: 'progress-bar--high-target',
+                test: progressBarTest({
+                    'UK': { target: 10000000, textTarget: '10 million' },
+                    'AU': { target: 750000 },
+                    'US': { target: 1000000 }
+                }, { showIncompleteLabelInsideProgressBar: true })
+            },
+            {
+                id: 'progress-bar--low-target',
+                test: progressBarTest({
+                    'UK': { target: 5000000, textTarget: '5 million' },
+                    'AU': { target: 300000 },
+                    'US': { target: 250000 }
+                })